flowetl/flowetl/flowetl/util.py

```
@@ -373,27 +521,54 @@ def create_dag(
+                    add_indexes,
+                ]
+                >> analyze
+                >> attach
+                >> latest_only
+                >> analyze_parent
+            )
+
+        with TaskGroup(
+            group_id="final_qa_checks", prefix_group_id=False
+        ) as final_qa_checks_group:
+            final_checks = (
```

Flake8 (7.2.0)

[error] local variable 'final_checks' is assigned to but never used

(F841)